From ac3e5aeb3f251ad377b94f0b5c2442567b48305a Mon Sep 17 00:00:00 2001
From: Ryan Thompson <ryanrthompson10@gmail.com>
Date: Tue, 2 Apr 2024 13:05:50 -0500
Subject: [PATCH 3/9] doc: updated default tuning mode to host

---
 host/libraries/libbladeRF/include/libbladeRF.h | 14 +++++---------
 1 file changed, 5 insertions(+), 9 deletions(-)

diff --git a/host/libraries/libbladeRF/include/libbladeRF.h b/host/libraries/libbladeRF/include/libbladeRF.h
index 4c871965..6fc323f1 100644
--- a/host/libraries/libbladeRF/include/libbladeRF.h
+++ b/host/libraries/libbladeRF/include/libbladeRF.h
@@ -3590,19 +3590,15 @@ int CALL_CONV bladerf_trim_dac_read(struct bladerf *dev, uint16_t *val);
 /**
  * Frequency tuning modes
  *
- * ::BLADERF_TUNING_MODE_HOST is the default if either of the following
- * conditions are true:
- *   - libbladeRF < v1.3.0
- *   - FPGA       < v0.2.0
+ * The default tuning mode, `BLADERF_TUNING_MODE_HOST`, can be overridden by
+ * setting a BLADERF_DEFAULT_TUNING_MODE environment variable to `host` or `fpga`.
  *
- * ::BLADERF_TUNING_MODE_FPGA is the default if both of the following
- * conditions are true:
+ * ::BLADERF_TUNING_MODE_HOST is the default tuning mode.
+ *
+ * ::BLADERF_TUNING_MODE_FPGA requirements:
  *  - libbladeRF >= v1.3.0
  *  - FPGA       >= v0.2.0
  *
- * The default mode can be overridden by setting a BLADERF_DEFAULT_TUNING_MODE
- * environment variable to `host` or `fpga`.
- *
  * @note Overriding this value with a mode not supported by the FPGA will result
  *       in failures or unexpected behavior.
  */
-- 
2.39.2

