From f24abc356a30ecd7ba19d64fdd6458012df3729a Mon Sep 17 00:00:00 2001
From: Robert Ghilduta <robert.ghilduta@nuand.com>
Date: Fri, 12 Jul 2024 04:09:00 -0400
Subject: [PATCH 9/9] debian: update change

---
 debian/changelog | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 4dda228d..7cc39122 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+bladerf (2024.07) noble; urgency=medium
+
+  * 2024.07 Release 2024.07
+
+ -- Robert Ghilduta (Nuand LLC) <robert.ghilduta@nuand.com>  Fri, 12 Jul 2024 03:32:13 -0400
+
 bladerf (2021.10) unstable; urgency=low
 
   * Release for 2021.10 tag
-- 
2.39.2

